tssc/step_implementer.py

```diff
@@ -200,20 +201,40 @@ def write_results(self, results):
             Existing results file has invalid yaml or existing results file does not have expected
```

Comment:
The comment block above refers to the "step specific results file" on lines 191 and 196

Reply:
Oh good call out. Also this comment block becomes part of the pydocs, so it's more than just a comment!